File: app/src/main/java/net/mieczkowski/yelpbusinessexample/models/auth/YelpAuthRequest.java
package net.mieczkowski.yelpbusinessexample.models.auth;

import net.mieczkowski.yelpbusinessexample.BuildConfig;

/**
 * Created by Josh Mieczkowski on 10/19/2017.
 */

public class YelpAuthRequest {

    private String grantType = "client_credentials";

    private String clientID = BuildConfig.YELP_CLIENT_ID;

    private String clientSecret = BuildConfig.YELP_CLIENT_SECRET;

    public YelpAuthRequest() {
    }

    public String getGrantType() {
        return grantType;
    }

    public String getClientID() {
        return clientID;
    }

    public String getClientSecret() {
        return clientSecret;
    }
}

File: dal/src/main/java/net/mieczkowski/dal/services/authService/models/YelpAuthRequest.kt
package net.mieczkowski.dal.services.authService.models

import net.mieczkowski.dal.BuildConfig

/**
 * Created by Josh Mieczkowski on 9/11/2018.
 */
class YelpAuthRequest {

    val grantType = "client_credentials"

    val clientID = "" //BuildConfig.YELP_CLIENT_ID

    val clientSecret = "" //BuildConfig.YELP_CLIENT_SECRET
}